dpnp/dpnp_iface_manipulation.py

```diff
@@ -2055,7 +2055,7 @@ def require(a, dtype=None, requirements=None, *, like=None):
     return arr
 
 
-def reshape(a, /, newshape, order="C", copy=None):
+def reshape(a, /, shape=None, *, newshape=None, order="C", copy=None):
```

Comment:
We need to update `dpnp.ndarray.reshape` method also, since it was also changed in numpy